### PR TITLE
Button by index and data to input

### PR DESCRIPTION
**Pull request by VlaDFykeR**

### CODE REVIEW

File: pages/elements/buttonElement.py [hunk 1]
```diff
@@ -33,8 +33,15 @@ def click_by_css_with_wait(self, wait_time=10):
         """
         wait = WebDriverWait(self.driver, wait_time)
         element = wait.until(
-            EC.element_to_be_clickable(By.CSS_SELECTOR, self.selector)
+            EC.element_to_be_clickable(By.CSS_SELECTOR)
```
Comment (TanyaKalistruk) [inline pages/elements/buttonElement.py:36]: Why selector was removed here?

File: pages/elements/InputElement.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    def __init__(self, selector):
+        """
+            Method for class fields declaration.
+        """
+        self.driver = webdriver.Chrome()
+        self.selector = selector
+
+    def send_data_input_css(self, string):
+        """
+        Method for send 'data to input'
```
Comment (TanyaKalistruk) [inline pages/elements/InputElement.py:18]: Method for sending data to input field

File: pages/elements/InputElement.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        Class for send 'data to input'
+    """
+    def __init__(self, selector):
+        """
+            Method for class fields declaration.
+        """
+        self.driver = webdriver.Chrome()
+        self.selector = selector
+
+    def send_data_input_css(self, string):
```
Comment (TanyaKalistruk) [inline pages/elements/InputElement.py:16]: How about just `send_data`?

File: pages/home_page.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
+from pages.elements.buttonElement import ButtonElement
```
Comment (TanyaKalistruk) [inline pages/home_page.py:1]: Was this file added in this pull request?

File: pages/home_page.py [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+            :param index: Variable index should contain number which we need to enter (number of card to event).
+        """
+        self.driver.find_element(By.CSS_SELECTOR, self.HOME_PAGE_CARD_TO_EVENTS_LINK_CSS.format(index)).click()
+
+    def click_number_of_page_btn(self, index):
+        """
+            Method for click number of page.
+            :param index: Variable index should contain number which we need to enter (number of page).
+        """
+        self.driver.find_element(By.CSS_SELECTOR, self.HOME_PAGE_NUMBER_OF_PAGE_BTN_CSS.format(index)).click()
```
Comment (TanyaKalistruk) [inline pages/home_page.py:49]: If you updated our class ButtonElement with a method for clicking on the button by index, can we use it in this class and remove this method?

File: pages/home_page.py [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+            :param string: Variable string should contain text which we need to enter (text for 'keyword' field).
+        """
+        self.driver.find_element(By.CSS_SELECTOR, self.HOME_PAGE_KEYWORD_INP_CSS).send_keys(string)
+
+    def send_hashtags_input(self, string):
+        """
+            Method for input text in 'hashtags' field.
+            :param string: Variable string should contain text which we need to enter (text for 'hashtags' field).
+        """
+        self.driver.find_element(By.CSS_SELECTOR, self.MORE_FILTERS_MENU_HASHTAGS_INP_CSS).send_keys(string)
```
Comment (TanyaKalistruk) [inline pages/home_page.py:63]: Almost the same question for input fields, you have added a class for them

File: pages/elements/InputElement.py (hunk 3 above)
Comment (VlaDFykeR) [reply pages/elements/InputElement.py:16]: What do you mean ?
About naming method or remane parameter (string)?

File: pages/elements/buttonElement.py (hunk 1 above)
Comment (VlaDFykeR) [reply pages/elements/buttonElement.py]: fixed

File: pages/elements/InputElement.py (hunk 2 above)
Comment (VlaDFykeR) [reply pages/elements/InputElement.py]: done

File: pages/elements/buttonElement.py [hunk 7]
```diff
@@ -27,14 +27,19 @@ def click_btn_by_xpath(self):
         button = self.driver.find_element(By.XPATH, self.selector)
         button.click()
 
-    def click_by_css_with_wait(self, wait_time=10):
+    def hover_and_click_by_css(self, wait_time):
```
Comment (TanyaKalistruk) [inline pages/elements/buttonElement.py:30]: Let's return default timeout

File: pages/home_page.py (hunk 5 above)
Comment (VlaDFykeR) [reply pages/home_page.py]: yes, done

File: pages/home_page.py (hunk 6 above)
Comment (VlaDFykeR) [reply pages/home_page.py]: done

**Review by TanyaKalistruk — APPROVED** [review]

**Pull request merged**